src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedContainer.cs

```diff
@@ -132,6 +132,8 @@ internal override ImmutableArray<TypeWithAnnotations> TypeArgumentsWithAnnotatio
 
         internal sealed override bool IsInterpolatedStringHandlerType => false;
 
+        public sealed override bool HasDeclaredRequiredMembers => false;
```

Comment:
It seems inconsistent that 'HasDeclaredRequiredMembers' is public while 'IsRequired' is internal.

Reply:
Muscle memory, didn't meant to make it public.